demos/slack/README.md

```diff
@@ -0,0 +1,14 @@
+# Configure Slack
+
+Basic configuration of the [Slack Plugin](https://wiki.jenkins.io/display/JENKINS/Slack+Plugin)
+
+## Sample configuration
+
+```yaml
+unclassified:
+  slackNotifier:
+    baseUrl: https://workspace.slack.com/services/hooks/jenkins-ci/
```

Comment:
I wouldn’t add the base url in the default sample, that’s only for slack compatible apps, teamDomain is the important one for most people

Reply:
there's a PR on the slack plugin to make the wording better on the help message for it,
slack compatible means apps like mattermost,
people actually using slack should use teamDomain

Here's the example I added to the readme of the slack plugin:
https://github.com/jenkinsci/slack-plugin#configuration-as-code